prov/efa/src/rxr/rxr_ep.c

```diff
@@ -222,7 +222,7 @@ int rxr_ep_post_user_buf(struct rxr_ep *ep, struct rxr_rx_entry *rx_entry, uint6
  * @return	On success, return 0
  * 		On failure, return a negative error code.
  */
-int rxr_ep_post_prov_buf(struct rxr_ep *ep, uint64_t flags, enum rxr_lower_ep_type lower_ep_type)
+int rxr_ep_post_internal_rx_pkt(struct rxr_ep *ep, uint64_t flags, enum rxr_lower_ep_type lower_ep_type)
```

Comment:
It's unclear why we are calling this a pkt (rxr_ep_post_internal_rx_pkt), yet when the user provides it we call it 'buf' (rxr_ep_post_user_recv_buf)

Reply:
This is because the function `rxr_ep_post_user_recv_buf` convert user buffer into a RX packet, then post it to the device. I added a comment about its behavior at the function header and updated commit message.